src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketDelete.java

```diff
@@ -103,26 +103,18 @@ public void run(ActionContext context) throws Exception {
           String.format("Unable to access or create bucket %s. ", gcsPath.getBucket())
             + "Ensure you entered the correct bucket path and have permissions for it.", e);
       }
-      gcsPaths.add(new Path(gcsPath.getUri()));
-    }
-
-    FileSystem fs;
-    context.getMetrics().gauge("gc.file.delete.count", gcsPaths.size());
```

Comment:
Shouldn't delete this, you should still calculate the number of the deleted blobs and emit this metrics